### PR TITLE
feat(website): add default and empty state examples to filter group pattern 

### PR DESCRIPTION
**Pull request by shleewhite**
- [x] remove default and empty state images
- [x] add new interactive default and empty state examples
- [x] link to codesandbox versions
- [x] test helper functions
- [x] vrt of default and empty state examples

**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [ ] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

File: packages/paste-website/src/component-examples/filter-group/constants.ts [hunk 1]
```diff
@@ -0,0 +1,70 @@
+import type {TableDataRow} from './types';
+
+export const roomTypes = ['All', 'Group', 'WebRTC Go', 'Peer to Peer'];
```
Comment (SiTaggart) [inline packages/paste-website/src/component-examples/filter-group/constants.ts:3]: nit: constants tent to be all caps https://github.com/twilio-labs/paste/blob/main/packages/paste-website/src/constants.ts

File: packages/paste-website/stories/FilterGroup.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import {DefaultFilterGroupExample, EmptyStateFilterGroupExample} from '../src/component-examples/filter-group/Examples';
+
+export default {
+  title: 'Website/FilterGroupExamples',
+};
+
+export {DefaultFilterGroupExample, EmptyStateFilterGroupExample};
```
Comment (SiTaggart) [inline packages/paste-website/stories/FilterGroup.stories.tsx:7]: praise: Really love that we have stories for these examples

File: packages/paste-website/src/__tests__/filterGroup.test.tsx [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+  });
+
+  it('should exclude the date if it is greater than the filter', () => {
+    const oneWeekAgo = new Date();
+    oneWeekAgo.setDate(oneWeekAgo.getDate() - 7);
+
+    const result = filterByDateRange(oneWeekAgo, 'day');
+    expect(result).toBeFalsy();
+  });
+});
```
Comment (SiTaggart) [inline packages/paste-website/src/__tests__/filterGroup.test.tsx:77]: praise: love to see this

**Review by SiTaggart — COMMENTED** [review]
suggestion: if you wanted to make this super hot, some cypress tests proving your interactive examples worked would put the cherry and icing on the cake 🍰

**Review by SiTaggart — APPROVED** [review]

**Review by zahnster — APPROVED** [review]
Praise: This looks so great. Love the page and the interactive examples. On the tech side, very nice code organization and reuse, and excellent tests.
🚢 it.

**Pull request merged**